manual execution guide (#1669)
smartcontractkit/documentation: src/content/ccip/best-practices.mdx

If `extraArgs` are left empty, a default of _200000_ `gasLimit` will be set.

## Decoupling CCIP Message Reception and Business Logic

> We advise separating the reception of CCIP messages from the core business logic of the contract. Implementing 'escape hatches' or fallback mechanisms is recommended to gracefully manage situations where the business logic encounters issues. To explore this concept further, refer to the [Defensive example](/ccip/tutorials/programmable-token-transfers-defensive).

Reviewer: ```suggestion
As a best practice, separate the reception of CCIP messages from the core business logic of the contract. Implement 'escape hatches' or fallback mechanisms to gracefully manage situations where the business logic encounters issues. To explore this concept further, refer to the [Defensive Example](/ccip/tutorials/programmable-token-transfers-defensive) guide.
```
